packages/keycodes/README.md

```diff
@@ -92,6 +92,10 @@ Keycode for ESCAPE key.
 
 Keycode for F10 key.
 
+<a name="isAppleOS" href="#isAppleOS">#</a> **isAppleOS**
```

Comment:
Oh, now I realise this would get exported and documented. In that case I feel inclined to just remove the dev environment code, mark this as unstable, or somehow move the component to this package. `isAppleOS` doesn't really feel like it belongs in the `keycodes` package.

Reply:
I'm fine with moving `isAppleOS` to the shortcut keyboard component if that unblocks this PR.

Reply:
I copied the platform utility as it made most sense. I know it is a bit of duplication but this one is OK. Any other way would illogically couple two different packages.